### DISCUSSION TITLE
How do I get a wiki as home page?

### DISCUSSION THREAD

**Discussion by Plobli** [datenstrom/community#829] [Ask a question] [answered]
Hello, I love my tiny, self hosted wiki with your nice software, but I try to use my wiki page as my home page. I found this note in your wiki: "To show the wiki on the home page, go to your content folder and delete the 1-home folder." 

When i delete the 1-home folder, i get an error an my home page: `Server Error, Wikipages '/wiki/' does not exist!`

**Comment by PetersOtto** [comment, marked as answer]
Hello Plobli.

I haven't tried it with a wiki yet, but with a blog it has always worked for me.

To find the error, you could look at the log file.
`system/extensions/yellow-website.log`

Or you can activate the `CoreDebugMode` in the `system/extensions/yellow-system.ini.`

Please have a look to this site: https://datenstrom.se/yellow/help/troubleshooting

**Reply by Plobli** [reply]
I get it. In my page.md i had a link like [wikipages /wiki/ 0]. In the case, the /wiki/ doesn't exists, i have to remove all /wiki/ from the page.md. Now everything works fine.
